<!DOCTYPE html>
<html lang="en"><head><!-- saved from url=(0014)about:internet -->
<meta charset="utf-8" /><meta name="viewport" content="width=device-width" /><meta content="(C) Copyright 2005" name="copyright" /><meta content="(C) Copyright 2005" name="DC.rights.owner" /><meta content="concept" name="DC.Type" /><meta content="Variants tab" name="DC.Title" /><meta content="Ion Torrent Browser Dx" name="DC.subject" /><meta content="Ion Torrent Browser Dx" name="keywords" /><meta content="GUID-CB56876C-83B2-4396-9C53-62B682DE6CA7.html" name="DC.Relation" scheme="URI" /><meta content="XHTML" name="DC.Format" /><meta content="GUID-62571109-3082-4A0C-B935-D73EE5A63FFF" name="DC.Identifier" /><meta content="en" name="DC.Language" /><title>Variants tab</title><link rel="prev" href="GUID-53E30299-AA5B-433F-A4C4-6E05BBE78A48.html" /><link rel="next" href="GUID-7A583DA2-FFE3-43AB-BC76-97A6F74A6F97.html" /><link href="static/screen.css?vYqzNzBe" rel="stylesheet" /><link href="static/print.css?Of1595s7" media="print" rel="stylesheet" /><link href="static/help.ico" rel="shortcut icon" /><script src="static/head.js?qUF72lF0"></script></head><body id="GUID-62571109-3082-4A0C-B935-D73EE5A63FFF" data-approot=""><header><a href="index.html" id="logo-link"><img class="logo" src="GUID-E9DF9DD7-2132-4F7C-93DE-CF7D25CD9CB9-screen.svg" alt="Logo"/></a><h1>Torrent Suite™ Assay Development Software Help 
	 </h1></header><section id="center"><article><noscript><p>JavaScript must be enabled in order to use this site.</p><p>Please enable JavaScript in your browser and refresh the page.</p></noscript><div id="thumbnailDialog"></div><a name="GUID-62571109-3082-4A0C-B935-D73EE5A63FFF"></a>

<h1 class="title topictitle1">Variants tab</h1>
<div class="body conbody"> 
	 <p class="p">The 
		<span class="ph uicontrol" id="GUID-62571109-3082-4A0C-B935-D73EE5A63FFF__GUID-EDEB2F42-6A1F-487E-B60E-9A029D50C312_unique_5"><a name="GUID-62571109-3082-4A0C-B935-D73EE5A63FFF__GUID-EDEB2F42-6A1F-487E-B60E-9A029D50C312_unique_5"></a>Variants tab</span> in the 
		<span class="ph uicontrol" id="GUID-62571109-3082-4A0C-B935-D73EE5A63FFF__GUID-A68D8E16-5661-4035-9462-EE363F747D4C"><a name="GUID-62571109-3082-4A0C-B935-D73EE5A63FFF__GUID-A68D8E16-5661-4035-9462-EE363F747D4C"></a>Results Report screen</span> displays a table of variant results. The table has a different configuration and contains different columns depending on the assay type. In some configurations, the table is split into subtabs. Column descriptions are provided below. 
	 </p>
 
	 <p class="p">The 
		<span class="ph uicontrol" id="GUID-62571109-3082-4A0C-B935-D73EE5A63FFF__GUID-BDAC38E1-9678-4721-AECF-91B48198A780"><a name="GUID-62571109-3082-4A0C-B935-D73EE5A63FFF__GUID-BDAC38E1-9678-4721-AECF-91B48198A780"></a>Summary subtab</span> reports whether any Variants, Gene Fusions, or CNVs are detected. If present, the specific variants are listed. 
	 </p>
 
	 
<div class="tablenoborder"><a name="GUID-62571109-3082-4A0C-B935-D73EE5A63FFF__TABLE_889873D6DBAB4DF291DC8B12D54DB03B"></a><table border="1" cellpadding="4" cellspacing="0" class="table" frame="border" id="GUID-62571109-3082-4A0C-B935-D73EE5A63FFF__TABLE_889873D6DBAB4DF291DC8B12D54DB03B" rules="all" summary=""><thead align="left" class="thead"> 
			 <tr class="row"> 
				<th class="entry" id="d33754e39" valign="top" width="37.735849056603776%"> 
				  <p class="p">Column 
				  </p>
 
				</th>
 
				<th class="entry" id="d33754e45" valign="top" width="62.264150943396224%"> 
				  <p class="p">Description 
				  </p>
 
				</th>
 
			 </tr>
</thead>
 
		  <tbody class="tbody"> 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p"> Gene/Gene Symbol 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p"> The gene where the variant is located. The value is the information present in the last column of your target BED file. If the value is absent then the variant caller labels these positions as region#1…region#N . 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p"> HotSpot ID 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p"> The name of the hotspot as defined in the BED file (see 
					 <a class="xref" href="GUID-FACE9419-26E9-4CB4-8E7A-0EDD3D7BDB3F.html">Manage hotspot files</a>). 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p">Test Result 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p">The determination of the assay for the given variant. 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p">Locus 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p"> The allele position in the reference genome. 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p"> Type 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p"> The type of variation detected: SNV (single nucleotide variation)/MNV (multi-nucleotide variation), INS (insertion), or DEL (deletion). 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p"> Genotype 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p">The genotype at the locus. 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p"> Ref 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p"> The reference base(s). 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p">Var 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p">The variant base(s). 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p"> Var Freq 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p">The most likely frequency of the variant allele (see Note below). 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p">Quality score 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p"> Quality score is the relative probability of either the &quot;reference&quot; hypothesis interval [0,cutoff), or the &quot;variant&quot; hypothesis interval [cutoff,1], Phred-scaled (‑10*log10). It is a measure of how strong the evidence for the variant call is. A higher score means more evidence for the call. The quality scores are capped at 100. 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p"> Cov 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p"> The number of reads covering the position after down-sampling. 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p">Classification 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p"> The list of names from the associated classification sets. 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p"> Gene Isoform 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p"> The symbol of the gene isoform. 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p">Driver Gene 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p"> The gene that regulates expression of the gene fusion isoform. 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p"> Type 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p"> The type of variation detected (e.g., Fusion). 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p"> Read Count 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p"> The number of valid reads aligned to specific isoform sequence. 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p">Normalized Read Count 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p"> The ratio of read count and total fusion panel mapped reads. 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p">Imbalance Score 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p">The score calculated by subtracting the number of 5&#39; reads from the number of 3&#39; reads, and dividing the result by the sum of all expression control reads. 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p"> Copy Number 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p">The copy number as determined by the software algorithm. 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p">CNV Confidence 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p"> The confidence interval for the copy number estimate at 5% and 95% confidence. 
				  </p>
 
				</td>
 
			 </tr>
 
			 <tr class="row"> 
				<td class="entry" headers="d33754e39 " valign="top" width="37.735849056603776%"> 
				  <p class="p"># Amplicons 
				  </p>
 
				</td>
 
				<td class="entry" headers="d33754e45 " valign="top" width="62.264150943396224%"> 
				  <p class="p"> The number of amplicons associated with the CNV ID minus the amplicons tagged as being outliers. 
				  </p>
 
				</td>
 
			 </tr>
 
		  </tbody>
 
		</table>
</div>
 
	 <div class="note note" id="GUID-62571109-3082-4A0C-B935-D73EE5A63FFF__GUID-8BD6A03F-1DBB-45BF-B77C-8C16E0EFF9B0"><a name="GUID-62571109-3082-4A0C-B935-D73EE5A63FFF__GUID-8BD6A03F-1DBB-45BF-B77C-8C16E0EFF9B0"></a><span class="notetitle">Note:</span>  
		<p class="p">The variant frequency reported in the run report is the most likely variant frequency in the reads after corrections are made for probable errors. The variant caller software uses this corrected frequency and the uncertainty in the observations to calculate the probability that the variant frequency in the sample falls within defined intervals that can be set by the user. The user sets a threshold frequency for deciding between genotypes: this defines intervals [0,c)[c,1-c)[1-c,1] corresponding to the three diploid genotypes (reference, heterozygous, homozygous) respectively. “c” here is the minimum variant frequency set during the creation of the assay. The software returns the genotype corresponding to the interval with the highest probability of containing the variant frequency in the sample. In most cases, the reads are highly accurate and do not have a high probability of error. In such cases, the observed frequency and the most likely frequency are similar, and are contained within a single interval leading to a high quality genotype. In some cases, if very few reads are observed, or the error rate is high, the observed counts and the most likely counts can be different, and the uncertainty in the real sample frequency can be high. When this happens, it is often impossible to exclude a heterozygous population in the sample, and the genotype is assigned to be heterozygous as a significant portion of the probability falls in that interval. 
		</p>
 
	 </div>
 
  </div>
<div class="related-links">
<div class="familylinks">
<div class="parentlink"><strong>Parent topic:</strong> <a class="link" href="GUID-CB56876C-83B2-4396-9C53-62B682DE6CA7.html">View the Results Report</a></div>
</div>
</div>
</article><nav id="topnav"><div id="breadcrumbs"></div><div id="topbuttons"><a id="button-hide-navigation" href="#" title="Hide navigation"><span>Hide navigation</span></a><a id="button-print" href="#" title="Print"><span>Print</span></a><a id="button-print-all" title="Print All"><span>Print All</span></a></div></nav></section><div id="leftbar"><ul class="tabs"><li class="currentli"><a class="current toc" href="#"><span>Contents</span></a></li><li><a class="glossary" href="#"><span>Glossary</span></a></li><li><a class="search" href="#"><span>Search</span></a></li></ul><nav class="toc current"><div class="loader"><img src="static/loader.gif" alt="Loading" /></div></nav><nav class="glossary"></nav><nav class="search"><div class="results"><span>No search has been performed.</span></div></nav></div><form id="searchform"><input class="search" type="search" placeholder="Search" /><input class="submit" type="submit" value="Go" /></form><script src="static/body.js?TC2GotM2"></script><footer><table xmlns:suite="http://www.suite-sol.com/lifetech"><tr><td class="col2"><p> 
			  For Research Use Only. Not for use in diagnostic procedures.</p><p> 
		  </p></td><td class="col3"><p class="col3a"> 
		</p><p class="col3b">2015.10.28 09:20:64</p></td><td class="col4"><a href="legal.html"><img src="static/logo.png" class="corplogo"/></a></td></tr></table></footer></body></html>
