<!DOCTYPE html>
<html lang="en"><head><!-- saved from url=(0014)about:internet -->
<meta charset="utf-8" /><meta name="viewport" content="width=device-width" /><meta content="(C) Copyright 2005" name="copyright" /><meta content="(C) Copyright 2005" name="DC.rights.owner" /><meta content="concept" name="DC.Type" /><meta content="Underlying statistical model" name="DC.Title" /><meta content="GUID-528399C1-2D6D-441C-8153-A2A1DD2F7B6C.html" name="DC.Relation" scheme="URI" /><meta content="XHTML" name="DC.Format" /><meta content="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23" name="DC.Identifier" /><meta content="en" name="DC.Language" /><title>Underlying statistical model</title><link rel="prev" href="GUID-528399C1-2D6D-441C-8153-A2A1DD2F7B6C.html" /><link rel="next" href="GUID-FAC8E45A-B12B-4093-AABB-B23FDF33C2F9.html" /><link href="static/screen.css?vYqzNzBe" rel="stylesheet" /><link href="static/print.css?Of1595s7" media="print" rel="stylesheet" /><link href="static/help.ico" rel="shortcut icon" /><script src="static/head.js?qUF72lF0"></script></head><body id="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23" data-approot=""><header><a href="index.html" id="logo-link"><img class="logo" src="GUID-E9DF9DD7-2132-4F7C-93DE-CF7D25CD9CB9-screen.svg" alt="Logo"/></a><h1>Torrent Suite™ Assay Development Software Help 
	 </h1></header><section id="center"><article><noscript><p>JavaScript must be enabled in order to use this site.</p><p>Please enable JavaScript in your browser and refresh the page.</p></noscript><div id="thumbnailDialog"></div><a name="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23"></a>

<h1 class="title topictitle1">Underlying statistical model</h1>
<div class="body conbody"> 
	 <p class="p"> Reads are observed from a population with some (unknown) variant allele frequency. We evaluate the likelihood of each read under the assumption that the read comes from a &quot;reference&quot; allele or a &quot;variant&quot; allele, in the context of the ensemble of all reads. This allows the 
		<span class="ph" id="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23__GUID-8730844E-9B31-49B1-9BDE-A8251740E122_unique_16"><a name="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23__GUID-8730844E-9B31-49B1-9BDE-A8251740E122_unique_16"></a><span class="ph" id="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23__GUID-6167EBE6-32A1-4969-87EF-28EABF32406C_unique_16"><a name="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23__GUID-6167EBE6-32A1-4969-87EF-28EABF32406C_unique_16"></a>Torrent Suite™</span> Assay Development Software</span> to construct a quality measure (change in log-likelihood) for each read, given the variability observed across all relevant reads. This log-likelihood allows the software to evaluate whether reads evaluated as individuals by the basecaller as variants or reference are simply mistakes, or real support for reference or variant. Using a uniform likelihood framework allows the software to evaluate insertions, deletions, or SNPs in the same way, so that all variant classes are handled in the same way, and scores can be compared across variants. 
	 </p>
 
	 <p class="p"> This likelihood was then evaluated based on the fit of the measured signal values for each read when compared to the predicted signal values for each read. These signal values are used without correction for phase, as this reflects the real observational variation in the data rather than a transformed version. One important aspect of using the uncorrected data is that a variant at a single position may cause changes across many flows: both because of out-of-phase populations, and because the flows in which predicted signal occurs may differ. This generally provides very strong statistical support for SNPs, insertions, and deletions that cause such disruption to the flow positions of predicted signal. 
	 </p>
 
	 <p class="p"> 
	 </p>
 
	 <div class="fig"><a name="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23__IMAGE_A84BDB537F7F45EBBFB77DE714D1E8BF"></a><img class="image" id="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23__IMAGE_A84BDB537F7F45EBBFB77DE714D1E8BF" src="GUID-1815A611-86B3-473B-9A46-3265DD7E6317-screen.png" /><span>Forward strand measured intensities</span><div class="desc"><span> Measured intensities on the forward strand for the SNP G-&gt;A at chr8, 145741764 (zero based). The primary in-phase population affects the signal strongly at flows 76 and 77, showing a strong difference between states compared to the variability. Predicted values under each hypothesis are shown in cyan and blue. Reference calls are triangles, variant calls are ×&#39;s. Residuals shown below to more clearly show the size of the residuals under the reference predictions. Note the &quot;echo&quot; of out-of-phase population at flows 87 and 88, which also contribute to the evaluation of these reads. 
		</span></div></div> 
	 <div class="fig"><a name="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23__IMAGE_5833CB149DE549C4979412D940BC0F26"></a><img class="image" id="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23__IMAGE_5833CB149DE549C4979412D940BC0F26" src="GUID-C4F8F4A0-0F80-45A4-80DC-1268C81628B9-screen.png" /><span>Reverse strand measured intensities</span><div class="desc"><span> This is the exact same SNP on the reverse strand. On this strand, it is a T-&gt;C change in flows 153 and 157. Note that there are additional changes induced at flows 156 and 158 as the &quot;A&quot; signal changes its position in the variant case when compared to the reference case. This provides additional support for the variant or reference, not obtainable by inspecting the bases independently, or deconvolving the flow signal. 
		</span></div></div> 
	 <p class="p"> To evaluate this likelihood for a given measurement when compared to a prediction, the 
		<span class="ph" id="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23__GUID-8730844E-9B31-49B1-9BDE-A8251740E122_unique_16"><a name="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23__GUID-8730844E-9B31-49B1-9BDE-A8251740E122_unique_16"></a><span class="ph" id="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23__GUID-6167EBE6-32A1-4969-87EF-28EABF32406C_unique_16"><a name="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23__GUID-6167EBE6-32A1-4969-87EF-28EABF32406C_unique_16"></a>Torrent Suite™</span> Assay Development Software</span> uses a t-distribution (heavy tailed to resist outliers), with a mean and variance fitted to the data. The mean is generally near zero, as predictions are generally accurate, but the system allows a bias term for each strand to adapt the predictions to any small systematic deviations within the data. The software provides an initial precision estimate for the mean to downweight deviations from zero (prediction-precision), but average with the data in standard Bayesian fashion. The variance is estimated based on the predicted intensity: the system software fits the nonlinear trend of variance by intensity to the data in all flows relevant to the variant across the ensemble of reads , then uses the predicted variance for each relevant individual flow in each read for the likelihood evaluation. 
	 </p>
 
	 <p class="p"> 
	 </p>
 
	 <div class="fig"><a name="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23__IMAGE_AB73A089BB9A451BB594118F8A0EFDD3"></a><img class="image" id="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23__IMAGE_AB73A089BB9A451BB594118F8A0EFDD3" src="GUID-25C6B5FC-C1AB-4AFA-A3CE-D78B56BF2A9E-screen.png" width="288" /><span>Log-likelihood of variant versus reference</span><div class="desc"><span> The log-likelihood of each read under the reference or variant hypothesis, under the bias and variance fit to the data. As seen previously, the reverse strand reads in this case provide more statistical support per read than the forward strand (although both are quite well separated). 
		</span></div></div> 
	 <p class="p"> The software fits these latent variables of bias and variance, as well as the likely probability of each read being a variant or reference using an EM algorithm. Given the current bias, variance and variant allele frequency, the system can assign each read a responsibility for reference or variant. Using the responsibility (soft-classification), improved estimates for bias, variance, and the variant allele frequency are derived. Once the software has converged to appropriate bias and variance terms, the posterior distribution of variant frequency given the ensemble of reads are evaluated. 
	 </p>
 
	 <div class="fig"><a name="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23__IMAGE_743946894AE14FF781A7A2593EBAF058"></a><img class="image" id="GUID-B3C55A95-949E-4F16-BDD2-C3E3C0FCDD23__IMAGE_743946894AE14FF781A7A2593EBAF058" src="GUID-EE2E7B59-254D-4BF4-AA51-DBA662BC6B09-screen.png" width="288" /><span>Posterior distribution of variant frequency</span><div class="desc"><span> Posterior distribution of log-likelihood for the reference allele frequency of this SNP, scaled so the maximum density is 1.0. Note that while each read has extremely strong statistical support, sampling variability still causes some uncertainty in the inference of the allele frequency in the sample. 
		</span></div></div> 
  </div>
<div class="related-links">
<div class="familylinks">
<div class="parentlink"><strong>Parent topic:</strong> <a class="link" href="GUID-528399C1-2D6D-441C-8153-A2A1DD2F7B6C.html">Variant Quality Score definition</a></div>
</div>
</div>
</article><nav id="topnav"><div id="breadcrumbs"></div><div id="topbuttons"><a id="button-hide-navigation" href="#" title="Hide navigation"><span>Hide navigation</span></a><a id="button-print" href="#" title="Print"><span>Print</span></a><a id="button-print-all" title="Print All"><span>Print All</span></a></div></nav></section><div id="leftbar"><ul class="tabs"><li class="currentli"><a class="current toc" href="#"><span>Contents</span></a></li><li><a class="glossary" href="#"><span>Glossary</span></a></li><li><a class="search" href="#"><span>Search</span></a></li></ul><nav class="toc current"><div class="loader"><img src="static/loader.gif" alt="Loading" /></div></nav><nav class="glossary"></nav><nav class="search"><div class="results"><span>No search has been performed.</span></div></nav></div><form id="searchform"><input class="search" type="search" placeholder="Search" /><input class="submit" type="submit" value="Go" /></form><script src="static/body.js?TC2GotM2"></script><footer><table xmlns:suite="http://www.suite-sol.com/lifetech"><tr><td class="col2"><p> 
			  For Research Use Only. Not for use in diagnostic procedures.</p><p> 
		  </p></td><td class="col3"><p class="col3a"> 
		</p><p class="col3b">2015.10.28 09:20:64</p></td><td class="col4"><a href="legal.html"><img src="static/logo.png" class="corplogo"/></a></td></tr></table></footer></body></html>
